### ISSUE TITLE
Doesn't work at all [resolved]

### ISSUE THREAD

**Issue by digital-discuss** [SelfControlApp/selfcontrol#547]
I'm not sure what the reason could be, but 2.2.2, just doesn't block anything in Firefox, when using the existing blacklist. When I add a site with "add to blacklist", it does seem to work, at least for the session.

In Safari every site seems blocked. I use no VPN, but in Firefox I sandbox facebook (blocked) and google (not blocked) with their respective container. It's my impression that a blocked site is blocked from everywhere in the system, so I don't view that as related as of now.

**Comment by digital-discuss** [comment]
Update, 25 days later I try the same version 2.2.2 again and this time my black listed is indeed working. Since then Firefox have been updated to 66. It would seem that Firefox's Proxy settings could have been involved. I noticed I could bypass the hosts with that( I know SC isn't using that) . 

I get different results from different sites, some waiting indefinitely and some saying "the connection was reset" and some "Unable to connect".  But it's working!
